xbmc/interfaces/legacy/InfoTagVideo.cpp

```diff
@@ -176,5 +176,25 @@ namespace XBMCAddon
     {
       return infoTag->m_strTrailer;
     }
+
+    String InfoTagVideo::getArtist()
+    {
+      return StringUtils::Join(infoTag->m_artist, g_advancedSettings.m_videoItemSeparator);
+    }
+
+    String InfoTagVideo::getAlbum()
+    {
+      return infoTag->m_strAlbum;
+    }
+
+    int InfoTagVideo::getTrack()
+    {
+      return infoTag->m_iTrack;
+    }
+
+    int InfoTagVideo::getDuration()
```

Comment:
unsigned int

Reply:
Thanks! See e62a54e7917724d880021102d2fd49c70d5a9d04.
I also had to add support for unsigned int in PythonSwig.cpp.template, see 61e31eef0c2557e4dbe3064157fbdbfc677f4eac.